app/components/base/table/table.scss

```diff
@@ -31,7 +31,7 @@ bl-table {
         display: table-header-group;
 
         > tr {
-            cursor:pointer;
```

Comment:
hhm without the pointer does it still look alright for other tables? Maybe we should have a class for table that are selcatble or not and show the pointer then